spec/customs_info_spec.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+describe EasyPost::CustomsInfo do
+  describe '.create' do
+    it 'creates a customs info' do
+      customs_info = described_class.create(Fixture.basic_customs_info)
+
+      expect(customs_info).to be_an_instance_of(described_class)
+      expect(customs_info.id).to match('cstinfo_')
+      expect(customs_info.eel_pfc).to eq('NOEEI 30.37(a)')
+    end
+  end
+
+  describe '.retrieve' do
+    it 'retrieves a customs info' do
+      customs_info = described_class.create(Fixture.basic_customs_info)
+      retrieved_customs_info = described_class.retrieve(customs_info.id)
+
+      expect(retrieved_customs_info).to be_an_instance_of(described_class)
+      expect(customs_info.to_s).to eq(retrieved_customs_info.to_s)
```

Comment:
Another wrong order here.

Reply:
I've gone ahead and searched through the repo and found about a half dozen others that were out of order and fixed them. The retrieved ordering should no longer be a problem.